### PR TITLE
APF-886-Description field was redundant in header subtitle and body d…

### PR DESCRIPTION
**Pull request by SudharsanThumattiSathyamoorthy**
…escription. Removed description in header subtitle. Signed-off-by: Sudharsan Sathiamoorthy<ssathiamoort@vmware.com>

### CODE REVIEW

File: connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json [hunk 1]
```diff
@@ -4,10 +4,7 @@
       "id": "00000000-0000-0000-0000-000000000000",
       "creation_date": "1970-01-01T00:00:00Z",
       "header": {
-        "title": "Bitbucket Server PR - Feature/APF-651 add image and test auth test/Sudharsan Sathiamoorthy",
-        "subtitle": [
-          "Description - * APF-651-Added integration test for getImage and getAuthKey\r\n* APF-651-Added integration test for test auth for connectors"
-        ]
+        "title": "Bitbucket Server PR - Feature/APF-651 add image and test auth test/Sudharsan Sathiamoorthy"
```
Comment (RyanBard) [inline connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json]: If you're going to change this, make it more like what I did with Github/Gitlab a few days ago:

```
"title": "Bitbucket Server Pull Request",
"subtitle": [
    "{projectKey}/{repositorySlug} #{pullRequestId}"
]
```

* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/github-pr/src/main/java/com/vmware/connectors/github/pr/GithubPrController.java#L231
* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/github-pr/src/main/resources/cards/text.properties#L2
* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/gitlab-pr/src/main/java/com/vmware/connectors/gitlab/pr/GitlabPrController.java#L252
* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/gitlab-pr/src/main/resources/cards/text.properties#L2

Comment (SudharsanThumattiSathyamoorthy) [reply connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json]: Added subtitle to the header in the format ```{projectKey}/{repositorySlug} #{pullRequestId}```.

File: connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json [hunk 2]
```diff
@@ -77,9 +75,7 @@
       "creation_date": "1970-01-01T00:00:00Z",
       "header": {
         "title": "Bitbucket Server PR - APF-27-Test commit for stash connector./Sudharsan Sathiamoorthy",
```
Comment (RyanBard) [inline connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json]: This just seems like too much information, formatted awkwardly (the title from the PR could have anything in it and can be very long), that will be in the body of the card.

The card is going to ellipsis this anyway, so you'll see something like "Bitbucket Server PR - APF-27-Test co...".

I recommend taking out the title + author from the card title.

Comment (SudharsanThumattiSathyamoorthy) [reply connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json]: Removed title + author from the card header.

**Comment by RyanBard** [comment]
If you don't mind, change the APF-812 references in commits to APF-886.

Go ahead and "rebase -i" rewrite the commits.

**Comment by RyanBard** [comment]
@rworsnop the commit messages are fixed up, please take a quick look and merge if everything looks good.

**Pull request merged**